src/event/containers/PastEvents.tsx

```diff
@@ -44,32 +46,75 @@ interface PastEventsContainerProps {
 const PastEventsContainer: React.FC<PastEventsContainerProps> = (props) => {
   const { auth, events, attendance, fetchAttendance, fetchPastEvents } = props;
 
+  const [timeframe, setTimeframe] = useState('All Time');
+  const [shownEvents, setShownEvents] = useState<any[]>(events);
```

Comment:
You can define Event as an interface and use it instead of `any[]`, though that would require adding it before the Props, probably. However, types might be better left off in the future, when the membership portal API has its types package exposed.